### ISSUE TITLE
Topword has some strange space between charts

### ISSUE THREAD

**Issue by YiqingC** [WheatonCS/Lexos#422]
On Topword page, choose option "Compare each document against other class(es)", sometimes there are weird space between charts: 

![topworderror](https://cloud.githubusercontent.com/assets/15393424/16500725/dad8feee-3ed3-11e6-8ec0-36b5f07ca7f6.png)

Those space seem to happen when the title is too long to stay on one line.

**Comment by scottkleinman** [comment]
I'm afraid to pull right now because of possible merge conflicts, but it could be that the Bootstrap grid can handle this issue.

**Comment by scottkleinman** [comment]
I've wrapped the tables in Bootstrap grid columns, which will probably solve the problem. However, in the process, I disabled the `topwordTableDiv` style (it's still in there--I just changed the name so it won't be called). This may have disabled some styling features we want to keep, so I'm not going to close this issue until we've had a chance to test and check out what's missing. I have pushed the change if you want to test.
